pkg/master/controller.go

```diff
@@ -240,6 +238,39 @@ func (c *Controller) CreateOrUpdateMasterServiceIfNeeded(serviceName string, ser
 	return err
 }
 
+// EndpointReconciler knows how to reconcile the endpoints for the apiserver service.
+type EndpointReconciler interface {
+	// ReconcileEndpoints sets the endpoints for the given apiserver service (ro or rw).
```

Comment:
This does seem to be documenting the implementation and not the interface.
If you want to amend it, I would focus on fixing that :)

On Mon, Jun 6, 2016 at 4:33 PM, Andy Goldstein <notifications@github.com>
wrote:

> In pkg/master/controller.go
> <https://github.com/kubernetes/kubernetes/pull/26915#discussion_r65988045>
> :
>
> > @@ -240,6 +238,39 @@ func (c *Controller) CreateOrUpdateMasterServiceIfNeeded(serviceName string, ser
> >  	return err
> >  }
> >
> > +// EndpointReconciler knows how to reconcile the endpoints for the apiserver service.
> > +type EndpointReconciler interface {
> > +	// ReconcileEndpoints sets the endpoints for the given apiserver service (ro or rw).
>
> Will change if needed. Lmk
> … <#m_1155649936887638275_>
> On Monday, June 6, 2016, Paul Morie ***@***.***> wrote: In
> pkg/master/controller.go <#26915 (comment)
> <https://github.com/kubernetes/kubernetes/pull/26915#discussion_r65979052>>
> : > @@ -240,6 +238,39 @@ func (c *Controller)
> CreateOrUpdateMasterServiceIfNeeded(serviceName string, ser > return err >
> } > > +// EndpointReconciler knows how to reconcile the endpoints for the
> apiserver service. > +type EndpointReconciler interface { > + //
> ReconcileEndpoints sets the endpoints for the given apiserver service (ro
> or rw). @DirectXMan12 <https://github.com/DirectXMan12> <
> https://github.com/DirectXMan12> In your original PR, the IP argument was
> changed to an array -- does the new interface need to have that signature
> change? — You are receiving this because you authored the thread. Reply to
> this email directly, view it on GitHub <
> https://github.com/kubernetes/kubernetes/pull/26915/files/04ce042ff9cfb32b2c776f755cc7abc886b8a441#r65979052>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/AAABYv-P3P-iMyxTA_NRAMrpDefj00w-ks5qJJs3gaJpZM4IvXB4>
> .
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/26915/files/04ce042ff9cfb32b2c776f755cc7abc886b8a441#r65988045>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAnglu-6KfZZU1mmVTL0xgLKQ8DNCIP-ks5qJK5XgaJpZM4IvXB4>
> .
>